holidayextras/ui-toolkit
# Changing to use variables in LESS

#### What does this PR do?
- Moves basic colours out into a variables LESS file
- Changes the location of all favicon and manifest file locations
- Indents bullet points

#### What unit or integration tests does this PR have?
N/A

#### What selenium tests does this PR have?
N/A

#### How should a developer review this?
Check that the code seems sane
`npm install`
`grunt docs` will build the docs for you to view changes

#### How should this be manually tested?
- Run `npm install`
- Run `grunt docs` & check the example http://localhost:4000
- Ensure the Favicon appear at the top of your tab.
- Check links on homepage to ensure a range of Favicons are visible.

#### Any background context you want to provide?
Favicons are standard for all websites so we need to ensure they work perfectly.

#### What are the relevant tickets?
N/A

#### What gif best describes how you feel about this work?
![high five](http://media.giphy.com/media/J68F2RSOy33d6/giphy.gif)

---
#### Quality checklist (for PR author to complete BEFORE code review):

#### IP or Developer Review:

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
